src/FluentValidation.Tests/TestValidator.cs

```diff
@@ -30,4 +30,14 @@ public class TestValidator : InlineValidator<Person> {
 			}
 		}
 	}
+
+    public class TestValidatorWithRuleSet : InlineValidator<Person>

```

Comment:
I'm cringing at adding this class in because I can't figure out how to add a `RuleSet` to the normal `TestValidator`.